app/views/topics/show.html.erb

```diff
@@ -1,13 +1,27 @@
 <%= render "govuk_component/title", title: @taxon.title %>
 
 <div>
-  <ul>
-    <% if @taxon.children? %>
+  <% if @taxon.children? %>
+    <h1>Sub Topics</h1>
+    <ul>
       <% @taxon.child_taxons.each do |child_taxon| %>
         <li>
           <%= link_to(child_taxon["title"], "/topics#{child_taxon["base_path"]}" ) %>
         </li>
       <% end %>
-    <% end %>
-  </ul>
+    </ul>
+  <% end %>
+</div>
+
+<div>
+  <% if @tagged_content.length > 0 %>
```

Comment:
I think you could squash up this update to the view, listing the tagged content, up to commit 5f86d7d8a4e939516041c91ef6b7aa16432c102b. If you do that `@tagged_content` will be defined and used in the same commit, and the story of the change will flow better.

I'm happy for the CSS changes and the update to the startup script to remain in separate commits.